### PR TITLE
Users/jarus95/data locations migrate

### PR DESCRIPTION
**Pull request by Jarus95**

### CODE REVIEW

**Comment by Jarus95** [comment]
Closes #9

**Comment by Jarus95** [comment]
Closes #3

**Comment by Jarus95** [comment]
Closes deb coment yozilsa issue srazu yopilmas ekan. qachonki merge qilganizda yopilib ketar ekan  @JohnnySenior

File: CashOverFlow/Models/Locations/Location.cs
Comment (JohnnySenior) [inline CashOverFlow/Models/Locations/Location.cs:1]: @Jarus95 Adashmasam Location.cs faylda 12 chi qatorda name property ni nulliable bilan muammosi bormi?

Comment (JohnnySenior) [inline CashOverFlow/Models/Locations/Location.cs:1]: image
@Jarus95

**Review by JohnnySenior — APPROVED** [review]
@Jarus95 Menimcha unchalik katta muammo emas shunchaki nullale o'chiqligi uchun shekilli. Approve qilyapman

**Comment by JohnnySenior** [comment]
> Closes #3

File: CashOverFlow/Models/Locations/Location.cs
Comment (Jarus95) [reply CashOverFlow/Models/Locations/Location.cs:1]: ha nullable qilib qoyish kerak shakilli @JohnnySenior

**Pull request merged**